index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@900&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/c90d8a2f33.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="about.css">
+</head>
+<body>
+  <header>
+    <div class="social-head">
+      <a href="#"><i class="fa-brands fa-facebook-square"></i></a>
+      <a href="#"><i class="fa-brands fa-twitter-square"></i></a>
+      <a href="#">Logout</a>
+      <a href="#">School Page</a>
+      <a class="engo" href="#">English</a>
+    </div>
+    <div class="hamb">
+      <img id="ham" src="./imgs/menu_FILL1_wght700_GRAD200_opsz48.png" alt="Menu icon">
+      <img id="close" src="./imgs/close_FILL1_wght700_GRAD200_opsz48.png" alt="Close button">
+    </div>
+    <div class="pages" id="pages">
+      <a href="index.html"><img class="uon-logo-f" src="./imgs/uonlogo.png" alt="University of nairobi Logo"></a>
+      
+      <div class="pages-a">
+        <a class="about-me" href="about.html">About</a>
+        <a href="#">Program</a>
+        <a href="#">Join</a>
+        <a href="#">Sponsor</a>
+        <a href="#">News</a>
+        <a href="#">Campaign</a>
```

Comment:
- [ ] Kindly make sure  When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. 😄 
- [ ] Kindly make sure When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears. 😄 

- [ ] What about preventing the sub-background from scrolling when the navigation bar is open? 🤔  You can use an `over-flow: hidden;` CSS property to do this, which will prevent your mobile nav menu page from being hovered over, as shown in this image 👇🏻.